src/api/app/controllers/comments_controller.rb

```diff
@@ -31,13 +31,28 @@ def find_obj
         @header = { project: @obj.name }
       end
     elsif params[:request_number]
-      @obj = BsRequest.find_by(number: params[:request_number])
-      raise ActiveRecord::RecordNotFound, "Couldn't find Request with number '#{params[:request_number]}'" if @obj.nil?
-
-      @header = { request: @obj.number }
+      find_request_or_action
     else
       @obj = User.session!
       @header = { user: @obj.login }
     end
   end
+
+  def find_request_or_action
+    @obj = BsRequest.find_by!(number: params[:request_number])
+    @header = { request: @obj.number }
+    return unless params.key?(:parent_id)
+
+    parent_comment = Comment.find_by(id: params[:parent_id])
+    raise ActiveRecord::RecordNotFound, "Couldn't find the parent comment with ID #{params[:parent_id]}" if parent_comment.nil?
```

Comment:
I want to have a custom error message here, so this is why I didn't use `find_by!` for retrieving the comment.